### PR TITLE
DataPrep & Rulebook bugfixes

### PR DESCRIPTION
**Pull request by tonybach**
JIRAs:
- https://issues.cask.co/browse/CDAP-12412
- https://issues.cask.co/browse/CDAP-12521

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/ColumnsTab/ColumnsTab.scss [hunk 1]
```diff
@@ -118,6 +121,49 @@ $option-hover-bg-color: #eeeeee;
...
+        }
+
+        &.active,
+        &:active {
+          background-color: initial;
+
+          &:hover,
+          &:focus {
+            background-color: initial;
+            outline: 0;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/ColumnsTab/ColumnsTab.scss]: is this intentional, being nested inside `.active`?

Comment (tonybach) [reply cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/ColumnsTab/ColumnsTab.scss]: Seems like I don't need it with the `&:hover` and `&:focus` rules already defined on lines 131 to 135. Will push a commit to remove.

**Comment by tonybach** [comment]
With the changes so far, there's currently a bug in Wrangler in Hydrator (on Firefox only), where the height of the columns(s) in the right panel stretch to fill out the remaining height. This is because of the `height: 100%` that was added to `.columns-tab .columns-list .table`.

However, this change is needed because in Chrome, if the user only has one column on the right panel, the Clear All/Select all dropdown won't be shown in full, since the height of the table can't accommodate the full dropdown. Still need to find a way to solve both these problems.

**Comment by ajainarayanan** [comment]
@elfenheart Have updated the styling for columns tab dropdown menu. Could you take another stab and the code.

**Review by elfenheart — APPROVED** [review]
👍  code LGTM

**Review by bdmogal — APPROVED** [review]

**Pull request merged**